pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -622,10 +625,11 @@ func (c *ClusterConfig) NewNodeGroup() *NodeGroup {
 	return ng
 }
 
-// NodeGroup holds all configuration attributes that are
+// NodeGroup holds configuration attributes that are
 // specific to a nodegroup
 type NodeGroup struct {
 	Name string `json:"name" jsonschema:"required"`
+	// Specify [custom AMIs](usage/custom-ami-support/)
```

Comment:
We can say "Specify a custom AMI id, `auto` or `auto-ssm`"

Reply:
I want to cover this in a separate PR improving the enum support for these annotations